### PR TITLE
Add browser tfm to test projects that used to run when it derived from Unix

### PR DESCRIPTION
**Pull request by safern**
Even though some of these will throw PNSE, I would like them to be intentionally disabled on Browser instead of silently skipping test projects that no longer match a tfm because Browser does not derive from Unix anymore.

cc: @marek-safar @ericstj @ViktorHofer @akoeplinger @Anipik @steveisok

### CODE REVIEW

**Review by steveisok — APPROVED** [review]

File: src/libraries/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests.csproj [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <DefineConstants Condition="'$(TargetsWindows)' == 'true'">$(DefineConstants);TargetsWindows</DefineConstants>
     <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
-    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser</TargetFrameworks>
```
Comment (Anipik) [inline src/libraries/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests.csproj:6]: We throw PNSE for system.diagnostics.process on Browser. these tests should fail

**Comment by Anipik** [comment]
@safern System.Diagnostics.Process & some other are pnse on browser, but the tests seem to be passing, i dont see u disabling them for browser. am i missing something here ? shouldnt they throw and fail here ?

**Comment by safern** [comment]
They're disabled: https://github.com/dotnet/runtime/blob/master/src/libraries/tests.proj#L39

That is why I want to make this change, so that we intentionally remove configurations instead of missing running tests because we missed to add a configuration to a test project. So whenever we cleanup that list as part of: https://github.com/dotnet/runtime/issues/38422 the tests fail in PR if they're not updated accordingly.

**Review by Anipik — APPROVED** [review]

**Review by ViktorHofer — APPROVED** [review]

**Comment by ViktorHofer** [comment]
Merging as the other runs seem to be stuck.

**Pull request merged**